WordPress/gutenberg
# Display keyboard shortcut for command palette in site view

## What?
Append the keyboard shortcut to the tooltip that appears when hovering/focusing the Command Palette button in the site editor.

## Why?
1. It's a useful hint :)
2. Consistency with other buttons that have an associated keyboard shortcut e.g. "Save".

## How?
Make use of `shortcut` prop on the `Button` that invokes the Command Palette

## Testing Instructions
1. Open the Site Editor
2. Mouse over or focus the search icon
3. Observe the appended keyboard shortcut:

<img width="431" alt="Screenshot 2023-07-21 at 15 20 46" src="https://github.com/WordPress/gutenberg/assets/846565/3806c2ca-0971-416f-9424-a5bcd9e845aa">
